.eslintrc.js

```diff
@@ -15,8 +15,8 @@ const LICENSE_HEADER = `
  */
 `
 
-module.exports = {	
-  root: true,	
+module.exports = {
```

Comment:
You can do `git checkout main -- .eslintrc.js` that should cause the file to be reverted back to the original version from main. It will lose _all_ changes for that file though.